__tests__/app.test.js

```diff
@@ -1,16 +1,89 @@
 const pool = require('../lib/utils/pool');
 const setup = require('../data/setup');
-// const request = require('supertest');
-// const app = require('../lib/app');
+const request = require('supertest');
+const app = require('../lib/app');
+const { animals } = require('../lib/animals-data.js');
+const { plants } = require('../lib/plants-data.js');
 
-describe('backend-express-template routes', () => {
+describe('animals routes', () => {
   beforeEach(() => {
     return setup(pool);
   });
-  it('example test - delete me!', () => {
-    expect(1).toEqual(1);
+
+  it('/animals should return a list of animals', async () => {
+    const res = await request(app).get('/animals');
+    const expected = animals.map((animal) => {
+      return { id: animal.id, name: animal.name };
+    });
+    expect(res.body).toEqual(expected);
+  });
+
+  it('/animals/:id should return animal detail', async () => {
+    const res = await request(app).get('/animals/1');
+    const sugarGlider = {
+      id: '1',
+      name: 'Sugar Glider',
+      family: 'Petauridae',
+      funFact:
+        'When moving between trees, this animal uses its gliding membrane, found between its wrists and ankles. When flying, the animal is able to control and change the size and shape of the membrane by changing the position of its limbs. Meanwhile, the tail helps them manage the flight, acting as a rudder.',
+      url: 'https://animalia.bio/sugar-glider?collection=37',
+    };
```

Comment:
since you're importing ```animals-data.js``` it might be fun to use the ```animals``` array here rather than hard-coding